library/core/src/cell/once.rs

```diff
@@ -164,6 +164,41 @@ impl<T> OnceCell<T> {
         }
     }
 
+    /// Gets the mutable reference of the contents of the cell,
+    /// initializing it with `f` if the cell was empty.
+    ///
+    /// # Panics
+    ///
+    /// If `f` panics, the panic is propagated to the caller, and the cell
+    /// remains uninitialized.
+    ///
+    /// It is an error to reentrantly initialize the cell from `f`. Doing
+    /// so results in a panic.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::cell::OnceCell;
+    ///
+    /// let mut cell = OnceCell::new();
+    /// let value = cell.get_mut_or_init(|| 92);
+    /// assert_eq!(value, &92);
+    /// *value += 2;
+    /// assert_eq!(value, &94);
+    /// let value = cell.get_mut_or_init(|| unreachable!());
+    /// assert_eq!(value, &94);
+    /// ```
+    #[inline]
+    #[stable(feature = "once_cell", since = "1.70.0")]
```

Comment:
This cannot be stable since it is something new. Make a new unstable feature like `once_cell_get_mut` that can be used for both functions.

Reply:
Shall I open a new issue, or the issue attr can be omitted?

I'm trying to use `#[unstable(feature = "once_cell_get_mut")]` here.

Reply:
Updated.